docs/index.html

```diff
@@ -1,13 +1,37 @@
 {% extends 'application.njk' %}
 
 {% block layout %}
-  <main id="content" class="hxBox-md">
+  <main id="content" class="hxBox-md homepage">
```

Comment:
add a second id to this instead of an additional class

```html
<main id="content homepage" class="hxBox-md">
```

Reply:
class needed ^^